### ISSUE TITLE
Hyper parameters utilized for Yolov11 against Yolov8

### ISSUE THREAD

**Issue by eypros** [ultralytics/ultralytics#22423]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello, I am trying to find the hyper-parameters for Yolov8 and Yolov11 models. I couldn't find any specific clarification over this but I have the impression that they both use the same hyper-parameters. 

For example in a conversation for Yolov8 in [here][1] it seems that the default option is `auto` for the optimizer which is further explained as Adam for the first 10000 iterations and SGD after that. This means it's dataset and device specific. If a dataset contains 20000 samples and it is trained using 16 batch then this means 20000/16=1250 iter/epoch. Which means 10000/1250=8 epochs using Adam and SGD after that for the rest of the training.

So, do we know if this is the case for Yolov11 also? I tend to believe so, since they utilize the same code to a large degree but a confirmation would be great.

  [1]: https://github.com/ultralytics/ultralytics/issues/3360

**Comment by UltralyticsAssistant** [comment]
👋 Hello @eypros, thank you for your interest in Ultralytics 🚀! This is an automated response to help you get moving quickly—an Ultralytics engineer will also assist you here soon.

For general guidance, we recommend visiting the [Docs](https://docs.ultralytics.com/) where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and answers to common questions.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question (e.g., about hyperparameters for YOLO8 or YOLO11), please include:
-Exact `ultralytics` version and environment details (OS, CUDA, GPU, Python, PyTorch)
-The exact command or Python snippet you’re running
-Relevant training args (optimizer choice, batch, epochs, imgsz, LR schedule)
-Training logs showing the start and a few epochs
-Any config files or overrides used
Also verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Before proceeding, please upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to ensure you’re testing the current behavior:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

-**Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
Are you referring to pretrained models? Or are you asking about default hyperparameters during training?

**Comment by eypros** [comment]
Basically about default hyperparam during training as you stated. But I would be interested to know if the default parameters have any difference with the pretrained models' parameters also.

**Comment by Y-T-G** [comment]
The default parameters are the same for all training in Ultralytics. They're defined in `default.yaml`

https://github.com/ultralytics/ultralytics/blob/57c207477002e80a234eb64781bf0692151a2b51/ultralytics/cfg/default.yaml

For the hyperparameters used for YOLO11 training, you can find them here:
https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030
